__tests__/p2sh/tx-proposal-melt-tokens.test.js

```diff
@@ -77,7 +77,7 @@ describe('melt-tokens tx-proposal api', () => {
       .set({ 'x-wallet-id': walletId });
     expect(response.status).toBe(200);
     expect(response.body.success).toBe(false);
-    expect(response.body.error).toEqual('Not enough tokens to melt: 1000000000000 requested, 200 available');
+    expect(response.body.error).toContain('Not enough tokens to melt: 1000000000000 requested');
```

Comment:
Using the best utxo selection algorithm we don't return the amount available when we don't have enough funds. Should we create an issue to improve this in the lib?

Reply:
Maybe not on the lib, but on the headless itself.

I think it's not the responsibility of utxo selection method to inform a token balance, but it's a useful information for the end user of the headless.